src/groupeddataframe/groupeddataframe.jl

```diff
@@ -24,11 +24,11 @@ mutable struct GroupedDataFrame{T<:AbstractDataFrame}
     parent::T
     cols::Vector{Symbol}                 # column names used for grouping
     groups::Vector{Int}                  # group indices for each row in 0:ngroups, 0 skipped
-    idx::Union{Vector{Int},Nothing}      # indexing vector sorting rows into groups
-    starts::Union{Vector{Int},Nothing}   # starts of groups after permutation by idx
-    ends::Union{Vector{Int},Nothing}     # ends of groups after permutation by idx
+    idx::Union{Vector{Int}, Nothing}      # indexing vector sorting rows into groups
+    starts::Union{Vector{Int}, Nothing}   # starts of groups after permutation by idx
+    ends::Union{Vector{Int}, Nothing}     # ends of groups after permutation by idx
     ngroups::Int                         # number of groups
-    keymap::Union{Dict{Any,Int},Nothing} # mapping of key tuples to group indices
+    keymap::Union{Dict{Any, Int}, Nothing} # mapping of key tuples to group indices
     lazy_lock::Threads.ReentrantLock     # lock is needed to make lazy operations
                                          # thread safe
```

Comment:
Need to adjust spacing to align comments.

Reply:
On it!